lib/decs.d.ts

```diff
@@ -0,0 +1 @@
+declare module "web-ext";
```

Comment:
What's `decs` stand for? Declarations?

Reply:
Yes "declarations". But its just an arbitrary name, I think this file can be named whatever you want (*.d.ts) (feel free to rename it).